tools/external_ic.F90

```diff
@@ -396,22 +395,16 @@ subroutine get_nggps_ic (Atm, fv_domain)
     call get_number_tracers(MODEL_ATMOS, num_tracers=ntracers, num_prog=ntprog)
     ntdiag = ntracers-ntprog
 
-!--- set the 'nestXX' appendix for all files using fms_io
-    if (Atm%grid_number > 1) then
-       write(gn,'(A4, I2.2)') "nest", Atm%grid_number
+    fname = 'INPUT/'//trim(fn_gfs_ctl)
```

Comment:
Is there a reason fn_gfs_ctl doesn't have INPUT/ prepended like the other fn_*_ic filenames?

Reply:
There really is no reason I can see as to why I wouldn't have just defined fn_gfs_ctl to include INPUT/ prepended.  I will address this in the next commit.